### PR TITLE
ENYO-5237: Add slider role and fix incorrect proptypes in SliderBehaviorDecorator

### PR DESCRIPTION
**Pull request by Baekwoo-s**
### Issue Resolved / Feature Added
An error message is being output due to incorrect proptypes.
TV don't read a11y because the slider role is missing.

### Resolution
Fixed incorrect proptypes (oneOf compares the values. oneOfType is correct.)
Added role="slider" in Slider component

### Links
ENYO-5237

### Comments
Enact-DCO-1.0-Signed-off-by: Baekwoo Jung (baekwoo.jung@lge.com)

### CODE REVIEW

File: packages/moonstone/Slider/SliderBehaviorDecorator.js [hunk 1]
```diff
@@ -159,6 +159,7 @@ const SliderBehaviorDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 			return (
 				<Wrapped
+					role="slider"
```
Comment (ybsung) [inline packages/moonstone/Slider/SliderBehaviorDecorator.js]: Please add `role` after `onFocus`.

**Comment by ybsung** [comment]
@Baekwoo-s 
Please add `Resolution` in the PR description.

File: packages/moonstone/Slider/SliderBehaviorDecorator.js (hunk 1 above)
Comment (Baekwoo-s) [reply packages/moonstone/Slider/SliderBehaviorDecorator.js]: @ybsung 
Usually the role is above the other prop.
Please see the code below.

https://github.com/enactjs/enact/blob/master/packages/moonstone/internal/SliderDecorator/SliderDecorator.js#L598
https://github.com/enactjs/enact/blob/release-2.0.0-beta.2-test/packages/ui/Button/Button.js#L174
https://github.com/enactjs/enact/blob/release-2.0.0-beta.2-test/packages/ui/Image/Image.js#L167

**Comment by ybsung** [comment]
Please update CHANGE LOG.md.

**Comment by ybsung** [comment]
LGTM

**Pull request merged**